package.json

```diff
@@ -7,30 +7,31 @@
     "build": "webpack --config webpack.prod.js",
     "preinstall": "bin/preinstall.js"
   },
-  "dependencies": {},
-  "devDependencies": {
+  "dependencies": {
     "bourbon": "~4.3",
     "bourbon-neat": "~1.9",
-    "copy-webpack-plugin": "^5.1.1",
-    "css-loader": "^3.1.0",
-    "file-loader": "^3.0.1",
     "font-awesome": "^4.7.0",
     "html5shiv": "^3.7.3",
-    "imports-loader": "^0.8.0",
     "jquery": "^3.4.1",
     "lato-font": "^3.0.0",
+    "roboto-fontface": "^0.10.0",
+    "wyrm": "^1.0.9"
+  },
```

Comment:
Raised in the other PR, I don't think this is necessarily correct. In the case that we wanted to make this available as an NPM/etc package, we shouldn't require users to install these dependencies when they wouldn't be used for anything outside the build process.